include/sysexits.h

```diff
@@ -35,9 +35,11 @@
 /*
  *  SYSEXITS.H -- Exit status codes for system programs.
  *
- *	This include file attempts to categorize possible error
- *	exit statuses for system programs, notably delivermail
- *	and the Berkeley network.
+ *	This include file attempts to categorize 4.4BSD error exit
```

Comment:
Hmmm, i know I asked this to be changed, then I deleted the comment as I fact checked it.

Up through 4.4BSD, it was used only by the mail subsystem. Nothing else used it.

So, I'm thinking that just deleting possible and not replace it with 4.4BSD.